include/APIClient/GPURequestor.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "restclient-cpp/restclient.h"
+#include "restclient-cpp/connection.h"
+
+#if defined(LUG_SYSTEM_WINDOWS)
+#pragma warning(push)
+#pragma warning(disable : 4003)
+#endif
+#include <json/json.hpp>
+#if defined(LUG_SYSTEM_WINDOWS)
+#pragma warning(pop)
+#endif
+
+#include "Router.hpp"
+
+using JSONResponse = std::tuple<int, nlohmann::json>;
```

Comment:
It's probably not the good file to put these `using` because you will use them in others classes for interacting with the API.